cff.el

```diff
@@ -305,10 +311,13 @@ headers or sources for FNAME."
                       (action . (lambda (candidate)
                                   (cff-process-one-found ,fname candidate))))))
                (helm :sources some-helm-source))
-           ;; otherwise use internal choice mode
-           (cff-choice-show found (lambda (candidate)
-                                    (cff-process-one-found fname candidate)
-                                    ))))))
+           (if (and cff-use-ivy-choice (fboundp 'ivy-read))
```

Comment:
here by introducing this variable I mentioned above we can use `cond` instead of `if` to make it more flexible